ui/ui-components/pages/model/[modelId]/edit.tsx

```diff
@@ -66,7 +61,7 @@ export default function Page(props) {
   const update = async (event) => {
     const _model = Object.assign({}, model);
     _model[event.target.id] = event.target.value;
-    setModel(_model);
+    setModelToUpdate(_model);
   };
 
   return (
```

Comment:
Looks like we need to move to use `modelToUpdate` in a couple places here - the auto-updating breadcrumb and title broke after this change (use in `ModelTabs`, `PageHeader`, `Head`)

Reply:
Yeah, I agree - though I bring it up because recently the same happened with the App edit page and a story was made to revert the behavior and make it match with all of our other edit pages. We should either do this everywhere or keep it consistent.

Reply:
@pedroslopez not a big fan but I updated it so that it's using the state on this page.